src/App.js

```diff
@@ -22,11 +23,17 @@ class App extends Component {
     return (
       <Router>
         <div>
-          <Banner />
-          <I18nBanner />
+          <Route path="/" render={props => (
+            <section>
+              <Banner />
+              <I18nBanner />
```

Comment:
Looks like the translation bar should always be fixed over the menu. This way the menu slide is also consistently positioned over where a user initially clicked to reveal the menu. Can we throw some CSS at this?

Reply:
> Looks like the translation bar should always be fixed over the menu. 
Are you referring to the mockups? Because I saw inconsistencies in how sidebars get displayed in them. I don't see a good reason functionally (or visually) to always show the language bar. Ok if we ask @cthibaultatx for clarification on this one?